src/schema/story.js

```diff
@@ -102,8 +103,12 @@ schema.method('clone', async function clone(user) {
 });
 
 schema.method('getPath', async function getPath() {
-  const advertiser = await connection.model('advertiser').findById(this.advertiserId);
-  return `story/${advertiser.slug}/${this.slug}/${this.id}`;
+  const [advertiser, publisher] = await Promise.all([
```

Comment:
The story host name will also need to be accounted for, correct?

Reply:
In reviewing the implementation I didn't see any reason to not use `domainName` for this purpose. The logic generating the story URLs already does this, and we don't refer to it as a CNAME anywhere. There is an account-level CNAME, which is different, and is the default `stories.account.com` domain, which is used if the publisher does not have a custom domain name.